modules/widgets/search.php

```diff
@@ -331,10 +331,7 @@ public function widget_non_instant( $args, $instance ) {
 		}
 
 		$title = isset( $instance['title'] ) ? $instance['title'] : '';
-
-		if ( empty( $title ) ) {
-			$title = '';
-		}
+		$title = ! empty( $title ) ? $title : '';
```

Comment:
Here I meant merging all of this (lines 333 to 337). Checking `empty()` also checks if `isset()`, so I don't think we need to do it twice here.

Reply:
Fixed!